### ISSUE TITLE
Citizens player NPCs are acting as normal players plugins are interacting with them

### ISSUE THREAD

**Issue by axemwa** [CitizensDev/Citizens2#1513]
I noticed that citizen NPCs are interacting with other plugins, going by what the console messages are showing.

Example: 

Worldguard plugin has command-on-entry flag set to feed a player when they walk into a region(spawn in this case), when this is set and an NPC is within that region console attempts to feed the NPCs with a response of error: Player not found. It's probably also due to the fact that the spawn is always loaded so when the console/server starts so do the command on entry flags for the NPCs.

This is causing many issues on my server and am requesting a fix please, I beleive it's also affecting a plugin called blockstatus which shows the online status of players by having a green(online) or red(offline) block autoplaced underneath a player npc depending on whether or not that player the npc is representing is online or not if that all makes sense.

Another plugin called customjoinitems is affecting citizen npcs where normally the plugin allows a custom item to be placed into a certain slot within the hotbar of an active player that can't be moved or dropped and it's purpose is for executing commands to open a menu and such. The customjoinitems plugin works normally and all, it's just that now each of those blockstatus npcs in that region are holding that custom item so it's like the system is registering the NPCs as real players?

Server log: https://www.dropbox.com/s/nx4mt1qe6pawq5d/latest.log?dl=0
Citizens config: https://hastebin.com/tasawevofo.scala
Minecraft version: 1.12.2
Citizens Version:  citizens-2.0.22-SNAPSHOT build #1547

**Comment by SXRWahrheit** [comment]
This sounds like those plugins need to utilize the Citizens API, rather than Citizens needing to do anything to support them.

**Comment by axemwa** [comment]
Good call, thankyou I just didn't know where to go about this should I forward this to their issue pages somehow if so how would I go about doing that give them the url to this page? Or should I just copy and paste what I entered here onto their issue page?

**Comment by SXRWahrheit** [comment]
https://github.com/CitizensDev/CitizensAPI

**Comment by SXRWahrheit** [comment]
http://jd.citizensnpcs.co/

**Comment by axemwa** [comment]
Do I forward those links to the authors?

**Comment by SXRWahrheit** [comment]
Do whatever you want. :)

I am not a Citizens dev, but as I understand it, it is not an issue that can be fixed by Citizens.
